Patches.py

```diff
@@ -2499,6 +2499,19 @@ def update_scrub_text(message: bytearray, text_replacement: list[str], default_p
 
     if world.settings.shuffle_individual_ocarina_notes:
         rom.write_byte(rom.sym('SHUFFLE_OCARINA_BUTTONS'), 1)
+        epona_notes = str(world.song_notes['Eponas Song'])
+        song_layout_in_byte_form = 0
+        if 'A' in epona_notes:
+            song_layout_in_byte_form |= 1 << 0
+        if '<' in epona_notes:
+            song_layout_in_byte_form |= 1 << 3
+        if '^' in epona_notes:
+            song_layout_in_byte_form |= 1 << 1
+        if 'v' in epona_notes:
+            song_layout_in_byte_form |= 1 << 2
+        if '>' in epona_notes:
+            song_layout_in_byte_form |= 1 << 4
```

Comment:
This is EXTRAORDINARILY nitpicky, but could these be put in order from 0 to 4? (or I'll just do it when I go to merge lol)